### ISSUE TITLE
Autocomplete addresses from Contacts

### ISSUE THREAD

**Issue by jancborchardt** [owncloud/maps#8]
When typing »Sander« into the search field (and assuming I have your address in ownCloud Contacts) I should get the suggestion »Sander, Somestreet, Somecity, Netherlands«.

This would be a great example of inter-app connection in ownCloud. If you want to see how it’s done, check out the mail app code: https://github.com/owncloud/mail (we do that for the »to« and »cc« fields)

**Comment by v1r0x** [comment]
I wrote a piece of code, which filters contact names and their addresses based on the search term. But I won't create a PR, since this is currently not good code and there are still peding PRs

**Comment by jancborchardt** [comment]
@v1r0x yeah, I’ll hope to get to it these days. :) The ownCloud conf was very busy. Hope you can make it next year, probably end of August again! https://owncloud.org/conf

**Comment by v1r0x** [comment]
@jancborchardt hopefully! There should be no thesis next year ;)
maybe you are at the 32c3 in december, too?
and don't hurry, I'm on holiday the next weeks. But I didn't want to spam more PRs with different master ;)

**Comment by jancborchardt** [comment]
@v1r0x if you just always branch PRs off master then it should be fine. :)
Not sure yet if I’ll come to C3 this year, I was at CCCamp though. Where are you based? Let me know if you happen to be in Munich any time. :)

**Comment by v1r0x** [comment]
@jancborchardt I know, but I continued master on my productive setup (I know this is not clever :D) and thus I have all my PRs already included. Will separate them after my holiday.
I'm close to Stuttgart, but I'm at Munich tonight for about 2 hours ;)

**Comment by jancborchardt** [comment]
@v1r0x hehe, yeah it’s indeed not clever. ;) Master should never be worked on, only in branches which can then be opened as pull requests.

**Comment by jancborchardt** [comment]
And yeah, see you later! :)
